soc/cdns/dc233c/Kconfig.defconfig

```diff
@@ -10,6 +10,9 @@ config XTENSA_MMU_NUM_L2_TABLES
 	int
 	default 48 if XTENSA_MMU
 
+config XTENSA_GEN_HANDLERS
+	default y
```

Comment:
i would select this in Kconfig file instead, because we have no other option, i.e. setting it to 'n' will fail after removing `_soc_inthandlers.h`

Reply:
Will do.  The idea/hope is really that this kconfig is a temporary compatibility tool.  Once we have "most" platforms using this then we can invert the default and remove all the settings except from "legacy" targets; once they're all using it we can remove it entirely.